internal/install/static_snapshot_yml.go

```diff
@@ -75,25 +75,41 @@ services:
       package-registry:
         condition: service_healthy
 
-  elastic-agent:
+  fleet-server:
     image: ${ELASTIC_AGENT_IMAGE_REF}
     depends_on:
       elasticsearch:
         condition: service_healthy
       kibana:
         condition: service_healthy
+    healthcheck:
+      test: "curl -f http://127.0.0.1:8220/api/status | grep HEALTHY 2>&1 >/dev/null"
+      retries: 12
+      interval: 5s
+    hostname: docker-fleet-server
+    environment:
+    - "FLEET_SERVER_ENABLE=1"
+    - "FLEET_SERVER_INSECURE_HTTP=1"
+    - "KIBANA_FLEET_SETUP=1"
+    - "KIBANA_FLEET_HOST=http://kibana:5601"
+    - "FLEET_SERVER_HOST=0.0.0.0"
+    ports:
+      - "127.0.0.1:8220:8220"
+
+  elastic-agent:
+    image: ${ELASTIC_AGENT_IMAGE_REF}
+    depends_on:
+      fleet-server:
+        condition: service_healthy
     healthcheck:
       test: "sh -c 'grep \"Agent is starting\" -r . --include=elastic-agent-json.log'"
```

Comment:
The `status` command landed, so it would be better to run that instead of this type of check.

`test: "./elastic-agent status"`

Should be enough, as it returns exit code 0 when the agent is healthy.

Reply:
Fixed